docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -79,7 +79,36 @@ To turn Cody off:
 
 ## Enable Cody only for some users
 
-To enable Cody only for some users, for example, when rolling out a Cody POC, follow all the steps mentioned in [Enabling Cody on your Sourcegraph instance](#enable-cody-on-your-sourcegraph-instance). Then, use the feature flag `cody` to turn Cody on selectively for some users. To do so:
+To enable Cody only for some users, for example, when rolling out a Cody POC, follow all the steps mentioned in [Enabling Cody on your Sourcegraph instance](#enable-cody-on-your-sourcegraph-instance). Then, do the following:
+
+<details>
+<summary>Sourcegraph 5.3+</summary>
+
+In Sourcegraph 5.3+, access to Cody is managed via user roles. By default, all users have access.
+
+First, ensure Cody is enabled in your site configuration. Go to **Site admin > Site configuration** (`/site-admin/configuration`) on your instance and set:
+
+```json
+    {
+      // [...]
+      "cody.enabled": true,
+      // Make sure cody.restrictUsersFeatureFlag is not in your configuration! If it is, remove it.
+    }
+```
+
+**Note:** Ensure `cody.restrictUsersFeatureFlag` is NOT in your site configuration. If it is, remove it or else the old feature-flag approach from Sourcegraph 5.2 and earlier will be used.
```

Comment:
```suggestion
<Callout type="note"> Ensure `cody.restrictUsersFeatureFlag` is **not** in your site configuration. If it is, remove it or else the old feature-flag approach from Sourcegraph 5.2 and earlier will be used.</Callout>
```

We have a component called `Callout` for `note`, `info`, `tip`, and `warning` types.